src/ansys/dpf/composites/constants.py

```diff
@@ -1,8 +1,17 @@
 """Collection of constants used across PyDPF Composites."""
-
 from enum import IntEnum
 
-__all__ = ("Spot", "Sym3x3TensorComponent", "FailureOutput")
+__all__ = (
+    "Spot",
+    "Sym3x3TensorComponent",
+    "FailureOutput",
+    "REF_SURFACE_NAME",
+    "FAILURE_LABEL",
+    "TIME_LABEL",
+)
+
+FAILURE_LABEL = "failure_label"
+TIME_LABEL = "time"
 
 
 class Spot(IntEnum):
```

Comment:
Please document if the indices must be in sync with the backend. Else you can ignore this comment.

Reply:
@roosre This indices are mapped in _get_rst_spot_index. The mapping there has to be always in sync with how the data is stored in the rst file (which will probably never change). So the enum is basically independent of the backend.

Reply:
The failure_label indicies are derived from the field name